tools/Get-PowerShellExtensionChangelog.ps1

```diff
@@ -372,7 +372,7 @@ function Get-PowerShellExtensionChangeLog {
     )
 
     $vscodePowerShell = Get-ChangeLog -LastReleaseTag $LastReleaseTag -Token $Token -HasCherryPick:$HasCherryPick.IsPresent -RepoUri 'https://api.github.com/repos/PowerShell/vscode-powershell' -RepoName 'vscode-PowerShell'
-    Push-Location ../PowerShellEditorServices
+    Push-Location (Join-Path $PSScriptRoot .. .. PowerShellEditorServices)
```

Comment:
We might want to mark this script as `#Requires -Version 6.0` since this Join-Path command won't work on <= v5.